internalsdk/auth/auth.go

```diff
@@ -52,21 +50,9 @@ type AuthClient interface {
 
 // NewClient creates a new instance of AuthClient
 func NewClient(baseURL string, userConfig func() common.UserConfig) AuthClient {
-	// The default http.RoundTripper is ChainedNonPersistent which proxies requests through chained servers
-	// and does not use keep alive connections. Since no root CA is specified, we do not need to check for an error.
-
-	var httpClient *http.Client
-
-	if baseURL == fmt.Sprintf("https://%s", common.APIBaseUrl) {
-		log.Debug("using proxied.Fronted")
-		//this is ios version
-		httpClient = &http.Client{
-			Transport: proxied.Fronted(30 * time.Second),
-		}
-	} else {
-		log.Debug("using proxied.ChainedNonPersistent")
-		rt, _ := proxied.ChainedNonPersistent("")
-		httpClient = pro.NewHTTPClient(rt, 30*time.Second)
```

Comment:
The effect of this (though it's not immediately obvious) was to prevent iOS clients from sending auth requests via the proxy.  Was there a reason for this (as opposed to trying the proxy, then falling back to domain fronting)?

Reply:
Yeah, there was a reason, Somehow using the same setting as Android does not work in IOS, Since we are not using Flashlight, not IOS. Take look here https://github.com/getlantern/engineering/issues/1625